src/assets/text/authors.js

```diff
@@ -0,0 +1,82 @@
+export default {
+    // do not delete section. delete individuals as needed. modify content as needed
+    // currently only 'fullName', 'firstName', 'initials', 'profile_link', and 'contribution' are used
+    primaryAuthors: [
+      {
+        firstName: 'Althea1',
+        lastName: 'Archer',
+        fullName: 'Althea Archer1',
+        initials: 'AA1',
+        profile_link: 'https://www.usgs.gov/staff-profiles/althea-archer',
+        role: 'lead developer',
+        contribution: 'led the design and development of the Vue website'
+      },
+      {
+        firstName: 'Althea2',
+        lastName: 'Archer',
+        fullName: 'Althea Archer2',
+        initials: 'AA2',
+        profile_link: 'https://www.usgs.gov/staff-profiles/althea-archer',
+        role: 'lead developer',
+        contribution: 'designed the illustrations'
+      },
+      {
+        firstName: 'Althea3',
+        lastName: 'Archer',
+        fullName: 'Althea Archer3',
+        initials: 'AA3',
+        profile_link: 'https://www.usgs.gov/staff-profiles/althea-archer',
+        role: 'lead developer',
+        contribution: 'made the charts'
+      }
+    ],
+    // do not delete section. delete any or all individuals as needed. modify content as needed
+    // currently only 'fullName', 'firstName', 'initials', 'profile_link', and 'contribution' are used
+    additionalAuthors: [
+      {
+        firstName: 'Cee',
+        lastName: 'Nell',
+        fullName: 'Cee Nell',
+        initials: 'CN',
+        profile_link: 'https://www.usgs.gov/staff-profiles/cee-nell',
+        role: 'supervisor',
```

Comment:
role is not used in the component

Reply:
Correct - I noted that in a comment. I figured it might be something we would like to incorporate.